zeek/assets/logs/zeek_tests.yaml

```diff
@@ -330,4 +330,111 @@ tests:
     tags:
      - "source:LOGS_SOURCE"
     timestamp: 1702360352763
-
+ -
+  sample: |-
+    {
+      "service" : "corelight",
+      "message" : "<134>Jan 12 18:13:46 machine-name {\"_path\":\"datared\",\"_write_ts\":\"2024-01-13T00:13:42.817478Z\",\"ts\":\"2024-01-13T00:13:42.817478Z\",\"conn_red\":1310,\"conn_total\":18460,\"dns_red\":1141,\"dns_total\":1604,\"files_red\":313,\"files_total\":796,\"http_red\":140,\"http_total\":140,\"ssl_red\":215,\"ssl_total\":227,\"weird_red\":20,\"weird_total\":20}"
+    }
```

Comment:
```suggestion
    <134>Jan 12 18:13:46 machine-name {"_path":"datared","_write_ts":"2024-01-13T00:13:42.817478Z","ts":"2024-01-13T00:13:42.817478Z","conn_red":1310,"conn_total":18460,"dns_red":1141,"dns_total":1604,"files_red":313,"files_total":796,"http_red":140,"http_total":140,"ssl_red":215,"ssl_total":227,"weird_red":20,"weird_total":20}
```

Reply:
Done, We have made this change but encountered the failure in validate-logs. So, Updated with that suggested change.